include/swift/AST/PluginRegistry.h

```diff
@@ -139,6 +139,8 @@ class PluginRegistry {
   /// Flag to dump plugin messagings.
   bool dumpMessaging = false;
 
+  std::mutex mtx;
```

Comment:
`getLoadedLibraryPlugins()` needs to lock this, too.

Reply:
I added a commit in this PR removing `PluginRegistry::getLoadedLibraryPlugins()`.